src/reaper_osara.cpp

```diff
@@ -667,6 +667,17 @@ void postCursorMovementScrub(int command) {
 		fakeFocus = FOCUS_RULER; // Set this even if we aren't reporting.
 }
 
+void postItemNormalize(int command) {
+	int selectedItemsCount = CountSelectedMediaItems(0);
```

Comment:
I know we say "0 items removed", "0 items added", etc., but those feel like they add clarity since they're potentially changing the number of items. In contrast, saying "0 items normalized" feels weird; I worry people will think something happened when it didn't. Can we just return or say "no selected items" if the count is 0?

Reply:
That makes sense. I went for "no selected items".